security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -10,11 +10,71 @@ The {cert-manager-operator} is a cluster-wide service that provides application
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10[cert-manager project release notes for v1.10].
+
+[IMPORTANT]
+====
+If you used the Technology Preview version of the {cert-manager-operator}, you must uninstall and move all related resources for the Technology Preview version before installing this version of the {cert-manager-operator}.
+
+For more information, see xref:../../security/cert_manager_operator/cert-manager-operator-uninstall.adoc#cert-manager-operator-uninstall[Uninstalling the {cert-manager-operator}].
+====
+
+[id="cert-manager-operator-1.10.2-new-features-and-enhancements"]
+=== New features and enhancements
+
+This is the general availability (GA) release of the {cert-manager-operator}.
+
+* The following issuer types are supported:
+** ACME
+** CA
+** Self-signed
+
+* The following ACME challenge types are supported:
+** DNS-01
+** HTTP-01
+
+* The following DNS-01 providers for ACME issuers are supported:
+** Amazon Route 53
+** Azure DNS
+** Google Cloud DNS
+
+// TODO: Any other features to list or things to call out as supported? proxy support?
+
+[id="cert-manager-operator-1.10.2-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `unsupportedConfigOverrides` field replaced user-provided arguments instead of appending them. Now, the `unsupportedConfigOverrides` field properly appends user-provided arguments. (*link:https://issues.redhat.com/browse/CM-23[CM-23]*)
++
+[WARNING]
+====
+Using the `unsupportedConfigOverrides` section to modify the configuration of an Operator is unsupported and might block cluster upgrades.
+====
+
+* Previously, the {cert-manager-operator} did not inject the necessary cluster wide egress proxy configuration. With this release, the {cert-manager-operator} now properly supports injecting custom CA certificates and detects cluster wide egress proxy configuration. (*link:https://issues.redhat.com/browse/CM-37[CM-37]*)
+
+* Previously, the {cert-manager-operator} was installed as a cluster Operator. With this release, the {cert-manager-operator} is now properly installed as an OLM Operator. (*link:https://issues.redhat.com/browse/CM-35[CM-35]*)
+
+[id="cert-manager-operator-1.10.2-known-issues"]
+=== Known issues
+
+* Using `Route` objects is not fully supported. Currently, {cert-manager-operator} integrates with `Route` objects by creating `Ingress` objects through the Ingress Controller. (link:https://issues.redhat.com/browse/CM-16[*CM-16*])
+
+* {cert-manager-operator} does not support using Azure Active Directory (Azure AD) pod identities to assign a managed identity to a pod. As a workaround, you can use a service principal to assign a managed identity. (link:https://issues.redhat.com/browse/OCPBUGS-8665[*OCPBUGS-8665*])
+
+* {cert-manager-operator} does not support using Google Kubernetes Engine (GKE) Workload Identity (link:https://issues.redhat.com/browse/OCPBUGS-9998[*OCPBUGS-9998*])
+
```

Comment:
@bergerhoffer since we have security/cert_manager_operator/cert-manager-operator-uninstall.adoc, but there is [OCPBUGS-9960](https://issues.redhat.com/browse/OCPBUGS-9960) about uninstall, and the bug can be easily hit by users in the user experience perspective, I suggest list this bug as known issue.

Reply:
@bergerhoffer , another bug about dns01, [OCPBUGS-8069](https://issues.redhat.com/browse/OCPBUGS-8069), which had Slack discussion link with PM/Dev, may need be listed as known issue

Reply:
The bug on the private DNS hosted zone need not be called under "Known issues" since this isn't really an issue it  is more of a invalid use case for DNS-01 type challenges (as mentioned by upstream as well). This limitation can be provided as a note in the DNS-01 section though.

Reply:
Thank you both.

When I tested uninstalling, I didn't see that checkbox option - does it only show in certain circumstances?

Either way, I noted it as a known issue. I've also added a note in the DNS-01 section saying that private DNS zones aren't supported.